src/api/global-api.md

```diff
@@ -193,6 +193,100 @@ const AsyncComp = defineAsyncComponent({
 
 **See also**: [Dynamic and Async components](../guide/component-dynamic-async.html)
 
+## resolveComponent
+
+:::warning
+`resolveComponent` can only be used within render functions.
+:::
+
+Allows resolving a `component` by its name, if it is available in the current application instance.
+
+Returns a `Component` or `undefined` when not found.
+
+```js
+const app = Vue.createApp({})
+app.component('MyComponent', {
+  /* ... */
+})
+```
+
+```js
+import { resolveComponent } from 'vue'
+setup() {
+  const MyComponent = resolveComponent('MyComponent')
+}
+```
+
```

Comment:
**question**: could we add arguments here if possible just like we did on directives?

```suggestion
#### Arguments

- `{ String } name
```